### PR TITLE
Added Indent & outdent Gantt locale added

### PR DESCRIPTION
**Pull request by karthikeyanraja**
Added locale text for indent and outdent for Gantt Chart control.

### CODE REVIEW

File: src/SfResources.ar-AE.resx [hunk 1]
```diff
@@ -810,6 +810,12 @@
   <data name="Gantt_ConfirmPredecessorDelete" xml:space="preserve">
     <value>هل أنت متأكد من أنك تريد إزالة رابط التبعية؟</value>
   </data>
+  <data name="Gantt_Indent" xml:space="preserve">
+    <value>مسافة بادئة</value>
+  </data>
+  <data name="Gantt_Outdent" xml:space="preserve">
+    <value>عفا عليها الزمن</value>
```
Comment (alansangeeths) [inline src/SfResources.ar-AE.resx:817]: Shows wrong meaning in google translate https://translate.google.co.in/?sl=ar&tl=en&text=%D8%B9%D9%81%D8%A7%20%D8%B9%D9%84%D9%8A%D9%87%D8%A7%20%D8%A7%D9%84%D8%B2%D9%85%D9%86&op=translate

File: src/SfResources.ar.resx [hunk 2]
```diff
@@ -666,6 +666,12 @@
   <data name="Gantt_ConfirmPredecessorDelete" xml:space="preserve">
     <value>هل أنت متأكد من أنك تريد إزالة رابط التبعية؟</value>
   </data>
+  <data name="Gantt_Indent" xml:space="preserve">
+    <value>مسافة بادئة</value>
+  </data>
+  <data name="Gantt_Outdent" xml:space="preserve">
+    <value>عفا عليها الزمن</value>
```
Comment (alansangeeths) [inline src/SfResources.ar.resx:673]: Shows wrong meaning in google translate https://translate.google.co.in/?sl=ar&tl=en&text=%D8%B9%D9%81%D8%A7%20%D8%B9%D9%84%D9%8A%D9%87%D8%A7%20%D8%A7%D9%84%D8%B2%D9%85%D9%86&op=translate

**Comment by alansangeeths** [comment]
hi @karthikeyanraja 
Which translater was used for this PR? Seems some language translation are incorrect.
Regards,
Alan Sangeeth S

**Comment by karthikeyanraja** [comment]
Hi team,

The changes were already updated in the source itself. Hence I'm closing this PR.

Regards,
Karthikeyan R

**Pull request closed without merging**